recipe/meta.yaml

```diff
@@ -21,14 +19,18 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
+    - nanobind >=1.9.2                       # [build_platform != target_platform]
+    - scikit-build                           # [build_platform != target_platform]
+    - cmake
     - {{ compiler('cxx') }}
     - {{ compiler('m2w64_cxx') }}      # [win and vc<14]
+    - {{ stdlib("c") }}        # [win and vc<14]
```

Comment:
If I understood https://conda-forge.org/news/2024/03/24/stdlib-migration/ correctly, there should be no `# [win and vc<14]` on this line I think.

Reply:
Sure, I'll take a look.